src/metabase/models/recent_views.clj

```diff
@@ -69,16 +72,30 @@
                 ;;       distinguish between them.
    :dashboard :table :collection])
 
-(defn- ids-to-prune-for-user+model [user-id model]
+(defn- moi->type
+  "Turns a model of interest into the base name of its table"
```

Comment:
nit — it's not the name of the table, right? aside from `collection`, but the others are different